runtime/onert/backend/cpu/DynamicTensorManager.cc

```diff
@@ -77,9 +77,17 @@ void DynamicTensorManager::changeShape(const ir::OperandIndex &ind, const ir::Sh
 
 void DynamicTensorManager::planDealloc(const ir::Operation *op, ir::OperandIndex operand_ind)
```

Comment:
How about Changing `const ir::Operation *` to `const ir::OperationIndex &`?

Reply:
@ragmai told me that `OperationIndex` is better for his work (control flow). I modified as he requested.